test/cppsuite/test_harness/workload/workload_validation.h

```diff
@@ -146,64 +147,65 @@ class workload_validation {
      * the test.
      */
     void
-    parse_schema_tracking_table(WT_SESSION *session, const std::string &collection_name,
-      std::vector<std::string> &created_collections, std::vector<std::string> &deleted_collections)
+    parse_schema_tracking_table(WT_SESSION *session, const std::string &tracking_table_name,
+      std::vector<uint64_t> &created_collections, std::vector<uint64_t> &deleted_collections)
     {
         WT_CURSOR *cursor;
         wt_timestamp_t key_timestamp;
-        const char *key_collection_name;
+        uint64_t key_collection_id;
         int value_operation_type;
 
         testutil_check(
-          session->open_cursor(session, collection_name.c_str(), nullptr, nullptr, &cursor));
+          session->open_cursor(session, tracking_table_name.c_str(), nullptr, nullptr, &cursor));
 
         while (cursor->next(cursor) == 0) {
-            testutil_check(cursor->get_key(cursor, &key_collection_name, &key_timestamp));
+            testutil_check(cursor->get_key(cursor, &key_collection_id, &key_timestamp));
             testutil_check(cursor->get_value(cursor, &value_operation_type));
 
-            debug_print("Collection name is " + std::string(key_collection_name), DEBUG_TRACE);
+            debug_print("Collection id is " + std::to_string(key_collection_id), DEBUG_TRACE);
             debug_print("Timestamp is " + std::to_string(key_timestamp), DEBUG_TRACE);
             debug_print("Operation type is " + std::to_string(value_operation_type), DEBUG_TRACE);
 
             if (static_cast<tracking_operation>(value_operation_type) ==
               tracking_operation::CREATE_COLLECTION) {
                 deleted_collections.erase(std::remove(deleted_collections.begin(),
-                                            deleted_collections.end(), key_collection_name),
+                                            deleted_collections.end(), key_collection_id),
                   deleted_collections.end());
-                created_collections.push_back(key_collection_name);
+                created_collections.push_back(key_collection_id);
             } else if (static_cast<tracking_operation>(value_operation_type) ==
               tracking_operation::DELETE_COLLECTION) {
                 created_collections.erase(std::remove(created_collections.begin(),
-                                            created_collections.end(), key_collection_name),
+                                            created_collections.end(), key_collection_id),
                   created_collections.end());
-                deleted_collections.push_back(key_collection_name);
+                deleted_collections.push_back(key_collection_id);
             }
         }
     }
 
     /* Update the data model. */
     void
-    update_data_model(const tracking_operation &operation, const std::string &collection_name,
+    update_data_model(const tracking_operation &operation, const uint64_t &collection_id,
```

Comment:
The intention behind a const ref is to avoid invoking the copy constructor and instead just copy a pointer. But this type is just an `uint64_t` anyway so there's no need to do this.